nsxt/nsxt_edge_details.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# coding=utf-8
+#
+# Copyright © 2015 VMware, Inc. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions
+# of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+# TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+# CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#__author__ = 'VJ49'
+
+import yaml
+import yamlordereddictloader
+from collections import OrderedDict
+
+import logging
+logger = logging.getLogger('vswitch')
+hdlr = logging.FileHandler('/var/log/chaperone/ChaperoneNSXtLog.log')
+formatter = logging.Formatter('%(asctime)s [%(levelname)s] %(funcName)s: %(message)s')
+hdlr.setFormatter(formatter)
+logger.addHandler(hdlr) 
+logger.setLevel(10)
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+        ),
+        supports_check_mode=True
+    )
+    final_dict = {}
+    main_list = list()
+    main_dict = {}
+    stream = open('/var/lib/chaperone/answerfile.yml', 'r')    
```

Comment:
This is nit-picky.  Allow passing in this file as a parameter to the module, but use this file as the default.  Look at `module_args` here https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html

Reply:
On that note, maybe we should use this module https://docs.ansible.com/ansible/latest/modules/include_vars_module.html to load the answers file into the the ansible playbook and pass whatever variables we need into the playbooks. Not sure how long this is going to take  but I believe it is the right approach and follow more of the Ansible standard

Reply:
Item added to the backlog for future release.